src/Microsoft.Health.Fhir.TemplateManagement/Models/DefaultTemplateInfo.cs

```diff
@@ -36,6 +38,7 @@ public DefaultTemplateInfo(DataType dataType, string imageReference, string temp
             { "microsofthealth/ccdatemplates:default", new DefaultTemplateInfo(DataType.Ccda, "microsofthealth/ccdatemplates:default", CcdaDefaultTemplatesResource) },
             { "microsofthealth/jsontemplates:default", new DefaultTemplateInfo(DataType.Json, "microsofthealth/jsontemplates:default", JsonDefaultTemplatesResource) },
             { "microsofthealth/stu3tor4templates:default", new DefaultTemplateInfo(DataType.Fhir, "microsofthealth/stu3tor4templates:default", Stu3ToR4DefaultTemplatesResource) },
+            { "microsofthealth/fhirtohl7v2templates:default", new DefaultTemplateInfo(DataType.Fhir, "microsofthealth/fhirtohl7v2templates:default", FhirToHl7v2DefaultTemplatesResource) },
```

Comment:
I think adding this here will imply theoretically $convert requests can use these templates. The processing will fail since the FhirToHl7v2 processor is not present there, but does this need to be added here for the new convert api flow as well?

Reply:
yeah the code DefaultTemplateCollectionProvider.cs is shared between the MCR fhirconverter and fhir-server and this is the only way to make the default templates work (or not work). It is pretty much all or nothing unless we wanted to modify the default template logic by runtime environment, or something like that, or have a config for the runtime that is passed in and chooses which templates to load. it is probably easiest to load all the templates and just not document the FhirToHl7v2 templates in the $convert docs

Reply:
But this map DefaultTemplateMap is only used in the ACR default template collection right? DefaultTemplateCollectionProvider.cs doesn't seem to be using this map specifically. Or am I missing something?

Reply:
hmm yes you are right. we can remove it from the map so it should not be able to be called from $convert, so I went ahead and removed it. The new template will still be loaded on startup but should not be able to be referenced now that it is removed from the map